pages/pipelines/input_step.md.erb

```diff
@@ -0,0 +1,223 @@
+# Input Step
+
+An *input* step is used to collect information from a user. 
```

Comment:
Given they're both going to exist and be documented next to each other, I think it's probably worth some heavy cross-promotion of the fact their options are identical -- so people can feel confident swapping between them as needed, without feeling obliged to re-check that all the options are still valid.